### DISCUSSION TITLE
Why URL Not Ignoring www?

### DISCUSSION THREAD

**Discussion by 534n0** [keepassxreboot/keepassxc#5972] [Q&A] [answered]
I'm storing my Google account login URL using https://www.google.com/ and I just realised today that it will not trigger **KeePassXC Browser Extension** when opening https://account.google.com/

I need to change the URL into https://google.com/ to match it. Is that the common behavior from it? Can it be improved? I think www is a common part from URL that can be ignored when analyzing & matching it with database.

**Comment by droidmonkey** [comment, marked as answer]
This is by design for very explicit reasons. www is a valid subdomain totally distinct from accounts or any other. There are many cases where ignoring an explicit url would cause the wrong credentials to be sent for people. We are considering ignoring the www but only to match the top level domain, not any sub domain.

**Reply by 534n0** [reply]
I guess the best alternative is to add https://google.com/ as additional URL. Am I right?

**Reply by droidmonkey** [reply]
Yes
